local.go

```diff
@@ -18,6 +18,14 @@ import (
 	"github.com/dedis/onet/network"
 )
 
+// LeakyTest is false by default, but can be set to true in `TestMain`
+// to have all tests undergo more stringent testing. This only influences
+// LocalTest.CloseAll
+//  0 - no leaky test at all
```

Comment:
Please use a typesafe version of this, i.e. `type LeakTestLevel int` and `const None iota LeakTestLevel` etc.

Reply:
Also removed the global variable, to force people to think how bad their code is if they have to write `local.CheckGoroutines` all over the place ;)